e2e-tests/cypress.config.ts

```diff
@@ -112,7 +112,10 @@ export default defineConfig({
 
       config.env.HAC_WORKSPACE = config.env.USERNAME.toLowerCase();
       if (config.env.PR_CHECK === true) {
-        config.env.HAC_NAMESPACE = `user-ns1`;
+        // According to the screenshot, currently there are two ns
+        // "user-ns1" and "user-ns2". Now we just enjoy the user-ns2.
+        // I guess we need to improve this part for hard-code ns.
+        config.env.HAC_NAMESPACE = `user-ns2`;
```

Comment:
The hardcoded Namespace is there just for the Konflux PR check - as we use locally deployed Konflux backend which has hardcoded user and namespace.